app/javascript/packs/containers/NavigationBar/CategoryContainer.jsx

```diff
@@ -0,0 +1,33 @@
+import {Link} from 'react-router-dom'
+import React from 'react'
+
+
+class CategoryContainer extends React.Component {
+  render() {
+    const {currentCategory, categories, remainingQuestions} = this.props;
+    const currentBuildingId = this.props.currentBuilding ? this.props.currentBuilding.id : null;
+    return (
+      <div>
+        <h1>{currentCategory ? currentCategory.name : null}</h1>
+        <div className={"categoryList"}>
+          {Object.keys(categories).map(id => {
+            return (
+              <p key={id}>
+                <Link
+                  to={`/buildings/${currentBuildingId}/edit/${categories[id].id}`}
+                  key={categories[id].id}>
+                  {categories[id].name} </Link>
+              </p>
+            )
+          })}
+        </div>
+        <div>
+          <p> {remainingQuestions} Questions Remaining</p>
+        </div>
+      </div>
+
+    )
+  }
+}
+
+export default CategoryContainer
```

Comment:
This appears to be a presentational component because it does not link to store, maybe put this in the components folder? Can make a `/Navigation` folder in the `/components` folder and put all of these presentational components in there